libraries/standard/mqtt/utest/mqtt_config.h

```diff
@@ -16,14 +16,14 @@
 
 /* Configure name and log level for the MQTT library. */
 #define LIBRARY_LOG_NAME     "MQTT"
-#define LIBRARY_LOG_LEVEL    LOG_INFO
+#define LIBRARY_LOG_LEVEL    LOG_NONE
 
 #include "logging_stack.h"
 
 /************ End of logging configuration ****************/
 
 /* Set network context to socket (int). */
-typedef int MQTTNetworkContext_t;
+typedef uint8_t** MQTTNetworkContext_t;
```

Comment:
I think that's part of a larger conversation regarding where the network context should be defined, but this it's in the test config to be consistent with the demo configs. I'm not sure how we could get around having this in a config file, since library headers use the `MQTTNetworkContext_t` type.